wordpress-mobile/WordPress-iOS
# Issue/9372 geo location not displayed correctly

Fixes #9372  

To test:
 - Set up your simulator or device to use French as the device language
 - Start the app 
 - Start a new post
 - Add a location to the post
 - Publish the post
 - refresh the post list
 - Edit the post again
 - Check that the location is set to the previous value.
